armadillo1/UPGRADE.md

```diff
@@ -0,0 +1,13 @@
+# Upgrading
+Here we talk about the upgrading of the Armadillo suite and the updating of the other parts of the system.
+
+### Upgrading the Operating System(OS)
+To keep the system up to date you will need to patch and/or update the system. This upgrading is <ins>NOT</ins> part of the Armadillo suite. It is the task of the owner of the server or the IT department to keep the system up to date and apply security patches. When an application of the Armadillo suite is fulnerable then we going to make sure that there will be a patch made and distributed to the systems involved.
```

Comment:
To keep the system up to date, you will need to patch and/or update the system. Upgrading is <ins>NOT</ins> part of the Armadillo suite. IT departments and server owners have the responsibility to keep their systems up to date and install security patches. If the Armadillo suite itself contains a vulnerability, a patch will be made and distributed. You need to install these patches to keep your system secure.